### PR TITLE
Updated notebooks after recent tests for CSHS

### PR DESCRIPTION
**Pull request by richardarsenault**
## Overview

This PR updates notebooks that were not compatible with the recent versions of xclim and ravenpy.

Also used the opportunity to clean the notebooks, removing all pre-run code, so they are all in a clean, new state.

### CODE REVIEW

File: docs/source/notebooks/03_Extract_geographical_watershed_properties.ipynb [hunk 1]
```diff
@@ -9,7 +9,9 @@
   },
```
Comment (cjauvin) [inline docs/source/notebooks/03_Extract_geographical_watershed_properties.ipynb:9]: <pre class="ql-syntax" spellcheck="false">Line #1.    str(latitude)
</pre><p>using <code>print</code>  (which does an implicit <code>str</code>  conversion) would probably work the same here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/422/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (cjauvin) [inline docs/source/notebooks/03_Extract_geographical_watershed_properties.ipynb:9]: <pre class="ql-syntax" spellcheck="false">Line #2.    grid=rio.open_rasterio(rasterio.MemoryFile(grid0[0]).open())
</pre><p>nitpick but use spaces around "=" for normal assigments.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/422/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (cjauvin) [inline docs/source/notebooks/03_Extract_geographical_watershed_properties.ipynb:9]: <pre class="ql-syntax" spellcheck="false">Line #7.    dem=rio.open_rasterio(rasterio.MemoryFile(dem0).open())
</pre><p>spaces around "="</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/raven/pull/422/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

**Comment by richardarsenault** [comment]
I just realized I had used the wrong initial versions before doing my changes. I have reimported the new ones and updated them accordingly, and now all outputs are preserved. Thanks for the feedback!

**Pull request merged**